tests/static/phpstan.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+	excludes_analyse:
+		- %currentWorkingDirectory%/src/Test/*
+	ignoreErrors:
+		- message: "#Strict comparison using === between bool and null will always evaluate to false.#"
```

Comment:
This seems like something we should actually fix, maybe an issue for our backlog

Reply:
This message is ignoring only in one class `File.php` as we suppress warning `@` of native php functions and PHPstan doesn't expect that function can return a null value.
I don't know how we can easily fix this.